package.json

```diff
@@ -19,9 +19,11 @@
     "eslint-plugin-prettier": "^4.2.1",
     "eslint-plugin-tsc": "^2.0.0",
     "husky": "^8.0.2",
+    "improved-yarn-audit": "^3.0.0",
```

Comment:
Not sure if necessary else remove all the other changes (lockfile, and added deps)

Reply:
Redundant dependency removed